### PR TITLE
Improve Debian commands in MAP update

### PR DESCRIPTION
**Pull request by smau-centreon**
## Description

Improve Debian commands in MAP update

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [x] 22.10.x (staging)
- [x] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-update.md [hunk 1]
```diff
@@ -32,9 +32,37 @@ sudo yum update centreon-map-engine centreon-map-web-client
...
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
+  
+ - Si MAP et MAP Legacy sont installés sur le même serveur :
+   
+   - Faites une sauvegarde du fichier **map.cnf** :
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-update.md:46]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-update.md [hunk 2]
```diff
@@ -32,9 +32,37 @@ sudo yum update centreon-map-engine centreon-map-web-client
...
+   - Mettez les paquets de **centreon-map-engine** à jour :
+   
+    ``` shell
+    sudo apt upgrade -o Dpkg::Options::="--force-overwrite" centreon-map-engine centreon-map-web-client
+    ```
+
+   - Récupérez la sauvegarde du fichier de configuration :
+   
+    ```shell
+    cp /etc/mysql/map.cnf.bk /etc/mysql/map.cnf
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-update.md:58]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf.bk /etc/my.cnf.d/map.cnf`

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-update.md [hunk 3]
```diff
@@ -32,9 +32,37 @@ sudo yum update centreon-map-engine centreon-map-web-client
...
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
+  
+ - Si MAP et MAP Legacy sont installés sur le même serveur :
+   
+   - Faites une sauvegarde du fichier **map.cnf** :
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-update.md:46]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-update.md [hunk 4]
```diff
@@ -32,9 +32,37 @@ sudo yum update centreon-map-engine centreon-map-web-client
...
+   - Mettez les paquets de **centreon-map-engine** à jour :
+   
+    ``` shell
+    sudo apt upgrade -o Dpkg::Options::="--force-overwrite" centreon-map-engine centreon-map-web-client
+    ```
+
+   - Récupérez la sauvegarde du fichier de configuration :
+   
+    ```shell
+    cp /etc/mysql/map.cnf.bk /etc/mysql/map.cnf
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-update.md:58]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf.bk /etc/my.cnf.d/map.cnf`

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/graph-views/map-web-update.md [hunk 5]
```diff
@@ -32,9 +32,37 @@ sudo dnf update centreon-map-engine centreon-map-web-client
...
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
+  
+ - Si MAP et MAP Legacy sont installés sur le même serveur :
+   
+   - Faites une sauvegarde du fichier **map.cnf** :
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/graph-views/map-web-update.md:46]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/graph-views/map-web-update.md [hunk 6]
```diff
@@ -32,9 +32,37 @@ sudo dnf update centreon-map-engine centreon-map-web-client
...
+   - Mettez les paquets de **centreon-map-engine** à jour :
+   
+    ``` shell
+    sudo apt upgrade -o Dpkg::Options::="--force-overwrite" centreon-map-engine centreon-map-web-client
+    ```
+
+   - Récupérez la sauvegarde du fichier de configuration :
+   
+    ```shell
+    cp /etc/mysql/map.cnf.bk /etc/mysql/map.cnf
```
Comment (cedricmeschin) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/graph-views/map-web-update.md:58]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf.bk /etc/my.cnf.d/map.cnf`

File: versioned_docs/version-22.04/graph-views/map-web-update.md [hunk 7]
```diff
@@ -14,30 +14,58 @@ Use the following procedure to update your MAP version:
...
-  ```
-  
-  </TabItem>
-  </Tabs>
+ - If MAP and MAP Legacy are installed on the same server:
+   
+   - Make a backup of the **map.cnf** file:
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```
Comment (cedricmeschin) [inline versioned_docs/version-22.04/graph-views/map-web-update.md:46]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`

File: versioned_docs/version-22.04/graph-views/map-web-update.md [hunk 8]
```diff
@@ -14,30 +14,58 @@ Use the following procedure to update your MAP version:
...
+   - Update the centreon-map-engine package
+   
+    ``` shell
+    sudo apt upgrade -o Dpkg::Options::="--force-overwrite" centreon-map-engine centreon-map-web-client
+    ```
+
+   - Retrieve the configuration file backup:
+   
+    ```shell
+    cp /etc/mysql/map.cnf.bk /etc/mysql/map.cnf
```
Comment (cedricmeschin) [inline versioned_docs/version-22.04/graph-views/map-web-update.md:58]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf.bk /etc/my.cnf.d/map.cnf`

File: versioned_docs/version-22.10/graph-views/map-web-update.md [hunk 9]
```diff
@@ -15,29 +15,57 @@ Use the following procedure to update your MAP version:
...
+ ```
   
-  </TabItem>
-  </Tabs>
+ - If MAP and MAP Legacy are installed on the same server:
+   
+   - Make a backup of the **map.cnf** file:
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```
Comment (cedricmeschin) [inline versioned_docs/version-22.10/graph-views/map-web-update.md:46]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`

File: versioned_docs/version-22.10/graph-views/map-web-update.md [hunk 10]
```diff
@@ -15,29 +15,57 @@ Use the following procedure to update your MAP version:
...
+   - Update the centreon-map-engine package
+   
+    ``` shell
+    sudo apt upgrade -o Dpkg::Options::="--force-overwrite" centreon-map-engine centreon-map-web-client
+    ```
+
+   - Retrieve the configuration file backup:
+   
+    ```shell
+    cp /etc/mysql/map.cnf.bk /etc/mysql/map.cnf
```
Comment (cedricmeschin) [inline versioned_docs/version-22.10/graph-views/map-web-update.md:58]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf.bk /etc/my.cnf.d/map.cnf`

File: versioned_docs/version-23.04/graph-views/map-web-update.md [hunk 11]
```diff
@@ -32,9 +32,37 @@ sudo dnf update centreon-map-engine centreon-map-web-client
...
+ ``` shell
+ sudo apt upgrade centreon-map-engine centreon-map-web-client
+ ```
+  
+ - If MAP and MAP Legacy are installed on the same server:
+   
+   - Make a backup of the **map.cnf** file:
+    
+    ```shell
+    cp /etc/mysql/map.cnf /etc/mysql/map.cnf.bk
```
Comment (cedricmeschin) [inline versioned_docs/version-23.04/graph-views/map-web-update.md:46]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf /etc/my.cnf.d/map.cnf.bk`

File: versioned_docs/version-23.04/graph-views/map-web-update.md [hunk 12]
```diff
@@ -32,9 +32,37 @@ sudo dnf update centreon-map-engine centreon-map-web-client
...
+   - Update the centreon-map-engine package
+   
+    ``` shell
+    sudo apt upgrade -o Dpkg::Options::="--force-overwrite" centreon-map-engine centreon-map-web-client
+    ```
+
+   - Retrieve the configuration file backup:
+   
+    ```shell
+    cp /etc/mysql/map.cnf.bk /etc/mysql/map.cnf
```
Comment (cedricmeschin) [inline versioned_docs/version-23.04/graph-views/map-web-update.md:58]: The correct path for map.cnf on Debian is `/etc/my.cnf.d/map.cnf`
so the backup copy should be `cp /etc/my.cnf.d/map.cnf.bk /etc/my.cnf.d/map.cnf`

**Review by cedricmeschin — CHANGES_REQUESTED** [review]

**Review by cedricmeschin — APPROVED** [review]

**Review by Jordan9131 — APPROVED** [review]

**Review by Jordan9131 — APPROVED** [review]

**Pull request merged**